### PR TITLE
feat: add Pricing page

### PR DESCRIPTION
**Pull request by vannyle**
This pull request introduces the Pricing pages.
- integrated the pricing form with HubSpot
- developed adaptive styles

**TODO**
- [ ] add animation to the hero section
- [x] review dropdown select option of Company size
- [x] put pricing link to menus

Preview: https://websitemain62807-pricing.gatsbyjs.io/pricing/

### CODE REVIEW

File: src/pages/pricing.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
```
Comment (sashasushko) [inline src/pages/pricing.jsx:2]:
```suggestion
import React from 'react';
```

File: src/pages/pricing.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+const Pricing = () => (
+  <Layout headerTheme="black" footerTheme="black" footerWithTopBorder>
+    <Hero />
+  </Layout>
+);
+
+export default Pricing;
+
+export const Head = ({ location: { pathname } }) => (
```
Comment (sashasushko) [inline src/pages/pricing.jsx:17]:
```suggestion
/* eslint-disable-next-line react/prop-types */
export const Head = ({ location: { pathname } }) => (
```

File: src/components/pages/pricing/hero/hero.jsx [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+    hutk: hubspotutk,
+    pageUri: href,
+  };
+
+  const onSubmit = async (data) => {
+    const { name, email, companyWebsite, companySize, message } = data;
+
+    setFormError('');
+    setFormState(FORM_STATES.LOADING);
+    try {
```
Comment (sashasushko) [inline src/components/pages/pricing/hero/hero.jsx:47]:
```suggestion
    setFormState(FORM_STATES.LOADING);

    try {
```

File: src/components/pages/pricing/hero/hero.jsx [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+      });
+
+      if (response.ok) {
+        doNowOrAfterSomeTime(() => {
+          setFormState(FORM_STATES.SUCCESS);
+          setFormError('');
+        }, 2000);
+      } else {
+        setFormState(FORM_STATES.ERROR);
+        setFormError('Something went wrong. Please reload the page and try again.');
```
Comment (sashasushko) [inline src/components/pages/pricing/hero/hero.jsx:82]:
```suggestion
        throw new Error('Something went wrong. Please reload the page and try again.');
```

**Comment by danieltprice** [comment]
A couple of wording suggestions:

"Are you interested in increasing your free tier limits or learning about pricing? Please complete the form below. A member of our sales team will be in touch."

"Interested in increasing your free tier limits or learning about pricing? Complete the form below to get in touch with our Sales team."

Also, do we want to solicit requests to increase "free tier limits" or to upgrade to a paid tier? If it's the later, perhaps the wording should be change to something like:

"Interested in increasing your limits with a Neon paid tier? Complete the form below to get in touch with our Sales team."

or 

"Interested in upgrading to a paid tier with greater limits? Complete the form below to get in touch with our Sales team."

**Review by danieltprice — DISMISSED** [review]
lgtm!

**Review by bolotskydev — APPROVED** [review]

**Pull request merged**